 gpio/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gpio/Makefile b/gpio/Makefile
index 85f5aa1..2882a1c 100644
--- a/gpio/Makefile
+++ b/gpio/Makefile
@@ -72,7 +72,7 @@ tags:	$(SRC)
 .PHONY:	install
 install: gpio
 	$Q echo "[Install]"
-	$Q cp gpio		$(DESTDIR)$(PREFIX)/bin
+	$Q cp gpio		$(DESTDIR)$(PREFIX)/bin/
 ifneq ($(WIRINGPI_SUID),0)
 	$Q chown root.root	$(DESTDIR)$(PREFIX)/bin/gpio
 	$Q chmod 4755		$(DESTDIR)$(PREFIX)/bin/gpio
